WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -47,6 +47,12 @@ extension UIImage {
             .applyTintColor(.placeholderImage)!
     }
 
+    /// Product categories Icon
+    ///
+    static var categoriesIcon: UIImage {
```

Comment:
when a new icon helper is created, we usually add a test case in [`IconsTests`](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/WooCommerceTests/Extensions/IconsTests.swift) - could we also add one for this new icon?

Reply:
fixed https://github.com/woocommerce/woocommerce-ios/pull/2043/commits/b1983f1b670fa66d63adcc71eb25bea4cd3ee3bc